installer_scripts/install-yb-voyager

```diff
@@ -540,10 +555,14 @@ get_passed_options() {
 				ONLY_PG="true"; 
 				shift 
 				;;
-			-v | --rebuild-voyager-local )
+			-r | --rebuild-voyager-local )
```

Comment:
@ShivanshGahlot I was talking about this change, I believe this changes the old arg `-v` to `-r` right? so the `OPTS` will change right?

Reply:
Ah okay. I did not intend to make this change but I guess made it while testing. I'd prefer reverting it back to the original